README.md

```diff
@@ -180,11 +180,23 @@ export default compose(
 )(DataHandler)
 ```
 
+### onMounted
+
+`onMounted` is executed when the component is mounted.
+
+### onUnMounted
+
+`onUnMounted` is executed when the component is unmounted.
+
+## Change Log
```

Comment:
I suggest using [Keep a Changelog format](https://keepachangelog.com/en/1.0.0/) which we also use in [eastern](https://github.com/wongnai/eastern/blob/master/CHANGELOG.md)

Reply:
Ok I fixed this one